code_of_conduct.html

```diff
@@ -0,0 +1,20 @@
+---
+permalink: /code-of-conduct.html
+layout: default
+---
+
+<div class="container">
+  <h2>Code Of Conduct</h2>
+
+  <h3>The Quick Version</h3>
+  <p>Our community is dedicated to providing a harassment-free experience for everyone, regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion (or lack thereof), or technology choices. We do not tolerate harassment in any form. Sexual language and imagery is not appropriate for any creations or community related communications. Community members violating these rules may be sanctioned or expelled from the community at the discretion of the community organisers.</p>
+
+  <h3>The Less Quick Version</h3>
+  <p>Harassment includes offensive comments related to gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion, technology choices, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, and unwelcome sexual attention.</p>
+  <p>Community members asked to stop any harassing behavior are expected to comply immediately.</p>
+  <p>Creators are also subject to the anti-harassment policy. In particular, creators should not use sexualised images, activities, or other material. Creators should refrain from creating a sexualised environment.</p>
+  <p>If a participant engages in harassing behavior, the community organisers may take any action they deem appropriate, including warning the offender or expulsion from the community.</p>
+  <p>If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a member of community staff immediately.</p>
```

Comment:
It's important to name a point of contact for concerns - that's a good first sign that we didn't just copy-paste a CoC here but are willing to enforce it. My first suggestion would be to make a coc@realm email alias and include it here.

Reply:
Thank you! 

I added the hello@ site email as the point of contact as I think single gates mean an easier journey for the users.
Would a coc@ display more credibility in your opinion?